src/nl/hannahsten/texifyidea/action/wizard/table/LatexTableWizardAction.kt

```diff
@@ -41,24 +35,16 @@ class LatexTableWizardAction : AnAction() {
             // Get the table information from the dialog, and convert it to latex.
             val tableTextToInsert = convertTableToLatex(dialogWrapper.tableInformation, indent)
 
-            // Use an insert action to insert the table.
-            InsertTable(tableTextToInsert).actionPerformed(file, project, editor)
-
-            // Insert the booktabs package.
-            WriteCommandAction.runWriteCommandAction(
-                project,
-                "Insert Table",
-                "LaTeX",
-                { file.psiFile(project)?.insertUsepackage(LatexPackage.BOOKTABS) },
-                file.psiFile(project)
-            )
+            return tableTextToInsert
```

Comment:
For example here, this class is named LatexTableWizardAction, but you removed the action, so now it's not an action anymore?

Reply:
I cannibalized it. With your blessing I can move it to a utility class (or dissolve it entirely)